source/features/unfinished-comments.tsx

```diff
@@ -28,7 +28,7 @@ function updateDocumentTitle(): void {
 
 	if (document.visibilityState === 'hidden' && hasDraftComments()) {
 		documentTitle = document.title;
-		document.title = '✏️ Draft - ' + document.title;
+		document.title = '✏️ Draft Comment - ' + document.title;
```

Comment:
Thanks for the PR!

I'd rather just drop "draft" since tabs are generally not that long

```suggestion
		document.title = '✏️ Comment - ' + document.title;
```

As a general feature behavior I'd love to eventually see the title _not_ depend on the tab focus.